pkg/package-server/storage/packagemanifest/reststorage.go

```diff
@@ -96,6 +98,22 @@ func (m *PackageManifestStorage) Get(ctx context.Context, name string, opts *met
 	return &manifest, nil
 }
 
+// Watch satisfies the Watch interface
+func (m *PackageManifestStorage) Watch(ctx context.Context, options *metainternalversion.ListOptions) (watch.Interface, error) {
+	// get namespace
+	namespace := genericapirequest.NamespaceValue(ctx)
+
+	// get selector
+	labelSelector := labels.Everything()
+	if options != nil && options.LabelSelector != nil {
+		labelSelector = options.LabelSelector
+	}
+
+	watcher := NewPackageManifestWatcher(namespace, labelSelector)
+	watcher.Run()
```

Comment:
It looks like before we call `watcher.Run()` we need to wire its channels to the provider using `provider.WatchPackageManifests(namespace string, out chan v1alpha1.PackageManifest, stop <-chan struct{})`.